docs/user/setup.rst

```diff
@@ -3,20 +3,20 @@
 Environment setup
 =================
 
-This is what you will be doing, in a nutshell:
+Scala Native has the following minimum system requirements:
 
-* installation of sbt
-* installation of LLVM and Clang
-* installation of native libraries
+* Java 8
```

Comment:
I'd say it would be more clear to `write Java 8+` or something along those lines.